exporter/signalfxexporter/metricdata_to_signalfxv2.go

```diff
@@ -162,8 +161,7 @@ func metricDataToSignalFxV2(
 						numDroppedTimeSeries++
 						logger.Warn(
 							"Timeseries for distribution metric dropped",
-							zap.Error(err),
-							zap.String("metric", *sfxDataPoint.Metric))
```

Comment:
Did you mean to drop "metric" log field?

Reply:
The field is also in the error, feels like duplicate information for me.

Reply:
Added back, will cleanup later.